app/src/main/java/com/sdp13epfl2021/projmag/database/impl/firebase/FirebaseCommentsDatabase.kt

```diff
@@ -0,0 +1,104 @@
+package com.sdp13epfl2021.projmag.database.impl.firebase
+
+import androidx.appcompat.app.AppCompatActivity
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.firestore.DocumentReference
+import com.google.firebase.firestore.FirebaseFirestore
+import com.sdp13epfl2021.projmag.database.interfaces.CommentsDatabase
+import com.sdp13epfl2021.projmag.database.interfaces.ProjectId
+import com.sdp13epfl2021.projmag.database.interfaces.UserdataDatabase
+import com.sdp13epfl2021.projmag.model.*
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+class FirebaseCommentsDatabase @Inject constructor(
+    private val firestore: FirebaseFirestore,
+    private val auth: FirebaseAuth,
+    private val userdataDatabase: UserdataDatabase
+) : CommentsDatabase,  AppCompatActivity() {
```

Comment:
This database should not be an activity.
```suggestion
) : CommentsDatabase {
```